docs/source/download.rst

```diff
@@ -73,13 +73,15 @@ ask us on Slack or report a bug by creating a new `github issue
 <https://github.com/firedrakeproject/firedrake/issues>`__. To help us
 diagnose what's going wrong, **please include the following log files**:
 
-* ``firedrake-install.log`` from Firedrake
-* ``configure.log`` and ``make.log`` from PETSc
+* ``firedrake-install.log`` from Firedrake, which you can find in the
+  directory where you invoked ``firedrake-install`` from
+* ``configure.log`` and ``make.log`` from PETSc, which you can find in
+  ``src/petsc/`` inside the directory where Firedrake virtual
+  environment was created
 
 Likewise, if it's ``firedrake-update`` that fails, please include the
-file ``firedrake-update.log``. The PETsc log files live in the
-directory ``src/petsc/`` inside the directory where the Firedrake
-virtual environment was created.
+file ``firedrake-update.log``. You can find this in the Firedrake
+virtual environment.
```

Comment:
You mean the firedrake install directory rather than the venv no?

Reply:
When I active the venv and run `firedrake-update`, I find the log file in the directory `$VIRTUAL_ENV`. Do you think it's clearer to say the install directory? I described them differently because `firedrake-install.log` will live in whatever directory you invoked the install script from, which is of course outside the venv.

Reply:
We should probably just fix that bug